src/projects/digital/api/circuit/src/internal/assembly/components/ConstantHighAssembler.ts

```diff
@@ -22,7 +22,7 @@ export class ConstantHighAssembler extends ComponentAssembler {
             {
                 kind: "SVG",
 
-                dependencies: new Set([AssemblyReason.TransformChanged]),
+                dependencies: new Set([AssemblyReason.TransformChanged, AssemblyReason.SelectionChanged]),
```

Comment:
just set `styleChangesWhenSelected` to true instead of adding this dependency (see other assemblers)